rpc/src/rpc.rs

```diff
@@ -4091,7 +4063,43 @@ fn rpc_perf_sample_from_perf_sample(slot: u64, sample: PerfSample) -> RpcPerfSam
     }
 }
 
-// RPC methods deprecated in v1.8
+pub mod rpc_deprecated_v1_18 {
```

Comment:
I forgot I wrote this code before v1.18 was branched. Rather than edit all these 18s, I'm going to propose backporting to v1.18 -- this is very low risk and small in scope, and backporting will allow us to remove this method altogether in v2.

Reply:
Should this also have been removed in #1886?

Reply:
I [included that at first](https://github.com/anza-xyz/agave/pull/1809#issue-2365198445), but realized we didn't deprecate the client methods in this PR. Still considering/hoping to remove in v2.0, but need to figure the best way to handle.